cmd/run.go

```diff
@@ -107,8 +107,19 @@ var RunCmd = &cobra.Command{
 				errScanner := bufio.NewScanner(stdErrPipe)
 				outScanner := bufio.NewScanner(stdOutPipe)
 				go func() {
+					//pure client app wait till daprd ready for conn
+					start_app := false
+					ready_msg := "dapr initialized. Status: Running"
 					for errScanner.Scan() {
-						fmt.Printf(print.Yellow(fmt.Sprintf("== DAPR == %s\n", errScanner.Text())))
+						text := errScanner.Text()
+						fmt.Printf(print.Yellow(fmt.Sprintf("== DAPR == %s\n", text)))
+						//stop searching after app started
+						if appPort == -1 && !start_app &&
+							strings.Index(text, ready_msg) != -1 {
```

Comment:
Checking for a string from Dapr output seems fragile. If this string changes in Dapr runtime, this would break. Its hard for anyone to track these string changes as its not an explicit interface.